### ISSUE TITLE
[MNG-6082] Introduction of model version 4.1.0

### ISSUE THREAD

**Issue by jira-importer** [apache/maven#7660]
**[Christian Schulte](https://issues.apache.org/jira/secure/ViewProfile.jspa?name=schulte77)** opened **[MNG-6082](https://issues.apache.org/jira/browse/MNG-6082?redirect=false)** and commented

---

**Remote Links:**
- [Page
](https://cwiki.apache.org/confluence/pages/viewpage.action?pageId=67636510)

1 votes, 4 watchers

**Comment by jira-importer** [comment]
**[Michael Osipov](https://issues.apache.org/jira/secure/ViewProfile.jspa?name=michael-o)** commented

This won't happen in 3.4, will it?

**Comment by jira-importer** [comment]
**[Christian Schulte](https://issues.apache.org/jira/secure/ViewProfile.jspa?name=schulte77)** commented

It won't pass the release vote. See [this mail](http://mail-archives.apache.org/mod_mbox/maven-dev/201610.mbox/%3CCA%2BnPnMwCxMGwRf5fgspetRWj894hLcgFdFPT_P8TDbgoG_7xaA%40mail.gmail.com%3E) on dev@.

**Comment by jira-importer** [comment]
**[Michael Osipov](https://issues.apache.org/jira/secure/ViewProfile.jspa?name=michael-o)** commented

Alright, removed fix version then.

**Comment by jira-importer** [comment]
**[Mark Struberg](https://issues.apache.org/jira/secure/ViewProfile.jspa?name=struberg)** commented

**edit because it was too hard worded**

just a general observation. Such a jira entry is extremely hard to read for any non-involved if it doesn't contain a summary about WHAT is supposed to get changed in 4.1.0!

This is as bad as a "TODO fixme" without any notion about what's wrong...

**Comment by jira-importer** [comment]
**[Christian Schulte](https://issues.apache.org/jira/secure/ViewProfile.jspa?name=schulte77)** commented

Hard worded? Pussy :-) It is totally bollocks and worth nothing now. See the 'Activity' tab what has changed recently. Plenty of issues were linked to this one. I removed those links because the issues were fixed without bumping the model version. I then wanted to delete this issue but couldn't. Closed it 'Won't Fix'. Someone with permissions can safely delete it.

**Comment by jira-importer** [comment]
**[Mark Struberg](https://issues.apache.org/jira/secure/ViewProfile.jspa?name=struberg)** commented

Well, I didn't want to offend anyone. It seems that a lot of work have been put into it. But nonetheless I was not able to find any answer to what this ticket is about without looking at all the linked tickets.

I think it's good to have as a summary ticket, but it should at least have a description. I've looked through the list of tickets and I could not find a ticket which requires us to bump the pom model version. There are quite a few underspecified areas (optional in dependencyManagement, etc) where I agree that it should get nailed better. But still nothing which requires a version bump imo. Can you point me to any new pom tag or attribute (and the corresponding ticket)? If there is none, then there is also no need to bump the version.

Re the dependencyManagement clarifications: we might need at least a compatibility flag in the maven CLI to allow older projects to still build with their 'broken' poms. Just an idea...

**Comment by jira-importer** [comment]
**[Christian Schulte](https://issues.apache.org/jira/secure/ViewProfile.jspa?name=schulte77)** commented

There is one last issue I will need to commit to master. After that, time has come to start working on the release notes and the website in general to document all of what is on master cleanly with examples. Once that is done, we have something to talk about. I need to finish that one last issue and then will need an awful lot of time documenting all of that. Once that documentation is in place, everyone can read about what is coming with 3.4 and this maybe will lead to further work on master. Without that documentation there is nothing to talk about. Takes time.
